module-cheatsheets/scRNA-seq-cheatsheet.md

```diff
@@ -37,23 +36,35 @@ Read the Base [`R` documentation](https://rdrr.io/r/).
 | Base `R`| [`prcomp()`](https://rdrr.io/r/stats/prcomp.html)| Principal Components Analysis|Executes a principal components analysis on specified matrix or data frame|
 | Base `R`| [`<-function(x) { <code> }`](https://adv-r.hadley.nz/functions.html) | Function | Creates a function that would take the defined parameters as input and execute the commands within the curly braces  |
 
+<br>
+
+### Salmon and `alevinQC`
+Read the command-line tool [Salmon documentation](https://salmon.readthedocs.io/en/latest/salmon.html).
+
+Read the R package [`alevinQC` documentation](https://rdrr.io/bioc/alevinQC/).
 
-### `SingleCellExperiment`, `DropletUtils`, `scran`, and `scater`
+Software/package | Piece of Code| What it's called| What it does |
+|--------------|-----------------|--------------|-----------------|
+| Salmon |[`salmon alevin`](https://salmon.readthedocs.io/en/latest/alevin.html)    | Salmon Alevin     | Runs the Alevin quantification from the command line  |
+|[`alevinQC`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html) | [`alevinQCReport()`](http://www.bioconductor.org/packages/devel/bioc//vignettes/alevinQC/inst/doc/alevinqc.html#generate-qc-report) | Alevin QC Report | Produces a QC (quality check) report from the `salmon alevin` output |
+
+
+<div style="page-break-after: always;"></div>
+
+### `SingleCellExperiment`, `S4Vectors`, `txmimeta`, and `DropletUtils`
```

Comment:
Do we need S4vectors in the header? I am not sure we do, though I can see the argument for keeping it.
```suggestion
### `SingleCellExperiment`, `txmimeta`, and `DropletUtils`
```